### PR TITLE
Pr/20

### LINKED ISSUE #16: [Feature Request] Use Miniart instead of Fanart for the Page
Because the titles are not shown, maybe you can use the Mini-Art from Jellyfin for the Pages.

### LINKED ISSUE #17: [ISSUE] Backgrounds are not showing in the title view
When you open the View of a title, there is no background of the title.

### PR DESCRIPTION
**Pull request by noahkamara**
resolves #16 
resolves #17

### CODE REVIEW

File: ABJC/UI/Subviews/Episode Row & Card/EpisodeCardView.swift [hunk 1]
```diff
@@ -15,13 +15,13 @@ extension LibraryView
         @EnvironmentObject var session: SessionStore
         
         /// Size
-        private var size: CGSize = CGSize(width: 548, height: 308.25 )
```
Comment (stephenb10) [inline ABJC/UI/Subviews/Episode Row & Card/EpisodeCardView.swift:18]: Might be best to change back to these dimensions, mine are rather large

**Review by stephenb10 — COMMENTED** [review]
Blur hashes are great, only issue is when the backdrop is very light, the white text on top is very hard to see.
Screen Shot 2021-05-19 at 8 21 36 am
It could be good to have a slight gradient form the backdrop to the normal background just before the text for readability.

**Comment by noahkamara** [comment]
> Blur hashes are great, only issue is when the backdrop is very light, the white text on top is very hard to see.
> Screen Shot 2021-05-19 at 8 21 36 am
> It could be good to have a slight gradient form the backdrop to the normal background just before the text for readability.

I updated the Blurhashes view to extract the brightness of the image and apply a filter if necessary (if brightness 150-255)

**Before:**
Detail View without effect

**After:**
Detail View with effect applied

File: ABJC/UI/Subviews/Episode Row & Card/EpisodeCardView.swift (hunk 1 above)
Comment (noahkamara) [reply ABJC/UI/Subviews/Episode Row & Card/EpisodeCardView.swift:18]: Yep. you're right. I reverted that by accident. I changed it back in the latest commit ed89bcad4acea77a5ad1932d5704587e87a26b64

**Pull request merged**